src/App.js

```diff
@@ -14,11 +14,76 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const icons = [
+  '🥟',
+  '🥕',
+  '🥚',
+  '🍦',
+  '🍏',
+  '🍞',
+  '🍥',
+  '🍯',
+  '🧉',
+  '🧄',
+];
+
+class App extends React.Component {
+  state ={
+    selectedGood: ['Jam'],
+  }
+
+  render() {
+    const { selectedGood } = this.state;
 
+    return (
+      <div className="App">
+        <h1 className="App__header">
+          {selectedGood.length > 0
+            ? (`${selectedGood} is selected`)
+            : 'No goods are selected'
+        }
+          {this.state.selectedGood.length > 0
+            && (
+            <button
+              className="removeAll-btn"
+              type="button"
+              onClick={() => {
+                this.setState({
+                  selectedGood: [],
+                });
+              }}
+            >
+              <span role="img" aria-label="removeAll">💣</span>
+            </button>
+            )}
+        </h1>
+        <ul className="list">
+          {goodsFromServer.map((good, i) => (
+            <li
+              key={good}
+              className={
+                selectedGood.includes(good)
```

Comment:
In future try to use 'classnames' library instead of ternary for switching classes